http-client/src/main/java/io/micronaut/http/client/netty/NettyClientHttpRequest.java

```diff
@@ -306,7 +307,7 @@ public StreamedHttpRequest toStreamHttpRequest() {
             String uriStr = resolveUriPath();
             io.netty.handler.codec.http.HttpMethod method = getMethod(httpMethodName);
             DefaultStreamedHttpRequest req = new DefaultStreamedHttpRequest(
-                    HttpVersion.HTTP_1_1, method, uriStr, (Publisher<HttpContent>) body);
+                                HttpVersion.HTTP_1_1, method, uriStr, false, (Publisher<HttpContent>) body);
```

Comment:
Similar question to the above, why is header validation disabled here?

Reply:
same answer as above